ecosystem/utils/submission_parser.py

```diff
@@ -27,35 +67,18 @@ def parse_submission_issue(body_of_issue: str) -> Repository:
 
     issue_formatted = mdformat.text(body_of_issue)
 
-    sections = defaultdict(
-        None, [_clean_section(s) for s in issue_formatted.split("### ")[1:]]
-    )
+    md_sections = issue_formatted.split("### ")[1:]
+    label_to_id = _get_label_to_id_map()
+    args = dict(_parse_section(s, label_to_id) for s in md_sections)
 
-    repo_name = sections["Github repo"].split("/")[-1]
-
-    name = repo_name
-    url = sections["Github repo"]
-    description = sections["Description"]
-    contact_info = sections["Email"]
-    alternatives = sections["Alternatives"]
-    licence = sections["License"]
-    affiliations = sections["Affiliations"]
-    website = sections["Website"]
-    if website == "_No response_":
-        website = None
-
-    labels = [l.strip() for l in sections["Labels"].split(",")]
-    if labels == ["_No response_"]:
-        labels = []
-
-    return Repository(
-        name=name,
-        url=url,
-        description=description,
-        licence=licence,
-        contact_info=contact_info,
-        alternatives=alternatives,
-        affiliations=affiliations,
-        labels=labels,
-        website=website,
-    )
+    args = {
+        field_id: (None if content == "_No response_" else content)
+        for field_id, content in args.items()
+    }
+
+    if args["labels"] is None:
+        args["labels"] = []
+    else:
+        args["labels"] = [l.strip() for l in args["labels"].split(",")]
```

Comment:
Nit: `l` looks ambiguous with 1

```suggestion
        args["labels"] = [x.strip() for x in args["labels"].split(",")]
```